### PR TITLE
Hiding password in IISWebAppManagementV2

### PR DESCRIPTION
**Pull request by 20shivangi**

### CODE REVIEW

File: Extensions/IISWebAppDeploy/Src/Tasks/IISWebAppMgmt/IISWebAppMgmtV2/AppCmdOnTargetMachines.ps1 [hunk 1]
```diff
@@ -438,26 +438,36 @@ function Update-AppPool
     {
         $userName = $appPoolCredentials.UserName
         $password = $appPoolCredentials.GetNetworkCredential().password
+        $hidingPassword = '***'
```
Comment (Ajay-MS) [inline Extensions/IISWebAppDeploy/Src/Tasks/IISWebAppMgmt/IISWebAppMgmtV2/AppCmdOnTargetMachines.ps1]: Remove this.

File: Extensions/IISWebAppDeploy/Src/Tasks/IISWebAppMgmt/IISWebAppMgmtV2/AppCmdOnTargetMachines.ps1 [hunk 2]
```diff
@@ -438,26 +438,36 @@ function Update-AppPool
...
         
         $appCmdArgs = [string]::Format('{0} -processModel.identityType:SpecificUser', $appCmdArgs)
         if (-not [string]::IsNullOrWhiteSpace($userName)) {
             $appCmdArgs = [string]::Format('{0} -processModel.userName:"{1}"',`
                                 $appCmdArgs, $userName)
+        $appCmdArgsLog = $appCmdArgs
```
Comment (Ajay-MS) [inline Extensions/IISWebAppDeploy/Src/Tasks/IISWebAppMgmt/IISWebAppMgmtV2/AppCmdOnTargetMachines.ps1]: remove this.

File: Extensions/IISWebAppDeploy/Src/Tasks/IISWebAppMgmt/IISWebAppMgmtV2/AppCmdOnTargetMachines.ps1 [hunk 3]
```diff
@@ -438,26 +438,36 @@ function Update-AppPool
...
         }
         if (-not [string]::IsNullOrWhiteSpace($password)) {
             $appCmdArgs = [string]::Format('{0} -processModel.password:"{1}"',`
                                 $appCmdArgs, $password)
+            $appCmdArgsLog = [string]::Format('{0} -processModel.password:"{1}"',`
```
Comment (Ajay-MS) [inline Extensions/IISWebAppDeploy/Src/Tasks/IISWebAppMgmt/IISWebAppMgmtV2/AppCmdOnTargetMachines.ps1]: Move this line above.

File: Extensions/IISWebAppDeploy/Src/Tasks/IISWebAppMgmt/IISWebAppMgmtV2/AppCmdOnTargetMachines.ps1 (hunk 1 above)
Comment (20shivangi) [reply Extensions/IISWebAppDeploy/Src/Tasks/IISWebAppMgmt/IISWebAppMgmtV2/AppCmdOnTargetMachines.ps1]: Done

File: Extensions/IISWebAppDeploy/Src/Tasks/IISWebAppMgmt/IISWebAppMgmtV2/AppCmdOnTargetMachines.ps1 (hunk 2 above)
Comment (20shivangi) [reply Extensions/IISWebAppDeploy/Src/Tasks/IISWebAppMgmt/IISWebAppMgmtV2/AppCmdOnTargetMachines.ps1]: done

File: Extensions/IISWebAppDeploy/Src/Tasks/IISWebAppMgmt/IISWebAppMgmtV2/AppCmdOnTargetMachines.ps1 (hunk 3 above)
Comment (20shivangi) [reply Extensions/IISWebAppDeploy/Src/Tasks/IISWebAppMgmt/IISWebAppMgmtV2/AppCmdOnTargetMachines.ps1]: Done

File: Extensions/IISWebAppDeploy/Src/Tasks/IISWebAppMgmt/IISWebAppMgmtV2/AppCmdOnTargetMachines.ps1 [hunk 4]
```diff
@@ -445,6 +445,8 @@ function Update-AppPool
                                 $appCmdArgs, $userName)
         }
         if (-not [string]::IsNullOrWhiteSpace($password)) {
+            $appCmdArgsLog = [string]::Format('{0} -processModel.password:"****"',`
```
Comment (Ajay-MS) [inline Extensions/IISWebAppDeploy/Src/Tasks/IISWebAppMgmt/IISWebAppMgmtV2/AppCmdOnTargetMachines.ps1]: Do we need to remove special character `

File: Extensions/IISWebAppDeploy/Src/Tasks/IISWebAppMgmt/IISWebAppMgmtV2/AppCmdOnTargetMachines.ps1 [hunk 5]
```diff
@@ -455,9 +457,14 @@ function Update-AppPool
     }
 
     $appCmdPath, $iisVersion = Get-AppCmdLocation -regKeyPath $AppCmdRegKey
-    $command = "`"$appCmdPath`" $appCmdArgs"
-
-    Write-Verbose "Updating application pool properties. Running command : $command"
+    
+    if($appCmdArgs -match "processModel.password") {
```
Comment (Ajay-MS) [inline Extensions/IISWebAppDeploy/Src/Tasks/IISWebAppMgmt/IISWebAppMgmtV2/AppCmdOnTargetMachines.ps1]: Follow pattern 

```
if (condition)
{
}
else 
{
}
```

File: Extensions/IISWebAppDeploy/Src/CHANGELOG.md [hunk 6]
```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.5.5] 24th July 2019
```
Comment (nschonni) [inline Extensions/IISWebAppDeploy/Src/CHANGELOG.md]:
```suggestion
## [1.5.6] 24th July 2019
```

Comment (20shivangi) [reply Extensions/IISWebAppDeploy/Src/CHANGELOG.md]: done

**Pull request merged**